properties/propertiesFile.go

```diff
@@ -56,10 +58,10 @@ type Models []*Model
 // A model/seed/snapshot schema
 type Model struct {
 	Name        string   `yaml:"name"`
-	Description string   `yaml:"description,omitempty"`
+	Description string   `yaml:"description"`
```

Comment:
including empty for these fields is helpful to match the existing behaviour of dbt. So that files are generated to the same standard!